planning/behavior_path_planner/include/behavior_path_planner/utils/geometric_parallel_parking/geometric_parallel_parking.hpp

```diff
@@ -121,10 +122,10 @@ class GeometricParallelParking
 
   void clearPaths();
   std::vector<PathWithLaneId> planOneTrial(
-    const Pose & start_pose, const Pose & goal_pose, const double R_E_r,
+    const Pose & start_pose, const Pose & goal_pose, const double R_E_far,
     const lanelet::ConstLanelets & road_lanes, const lanelet::ConstLanelets & shoulder_lanes,
-    bool is_forward, const double end_pose_offset, const double lane_departure_margin,
-    const double arc_path_interval);
+    bool is_forward, bool left_side_parking, const double end_pose_offset,
```

Comment:
const

Reply:
fixed in [const](https://github.com/autowarefoundation/autoware.universe/pull/4881/commits/9ac9482626b492e1ca1b5110c316f80282e92ed6)